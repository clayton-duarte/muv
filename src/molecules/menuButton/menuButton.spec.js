import React from 'react';

import MenuButton from '.';

it('should render a menuButton', () => {
  const wrapper = mount(
    <MenuButton />,
  );
  expect(wrapper).toMatchSnapshot();
});
