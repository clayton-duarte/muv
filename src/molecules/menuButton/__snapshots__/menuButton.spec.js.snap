// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a menuButton 1`] = `
<MenuButton
  open={false}
>
  <styled.button
    open={false}
    theme={
      Object {
        "animation": [Function],
        "color": Object {
          "black": "#222222",
          "darkBlue": "#002C5F",
          "darkGray": "#515151",
          "error": "#dd3333",
          "gray": "#a9abae",
          "lightGray": "#e5e5e5",
          "overlay": "rgba(0, 0, 0, .15)",
          "primary": "#00528F",
          "secondary": "#00245f",
          "success": "#33bb33",
          "warn": "#dd3333",
          "white": "#fefefe",
        },
        "font": Object {
          "color": "#222222",
          "family": "\\"Ubuntu\\", sans-serif",
          "size": "16px",
          "spacing": 0,
        },
        "shape": Object {
          "border": ".1rem solid rgba(0, 0, 0, .15)",
          "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
          "margin": ".25rem .125rem",
          "maxWidth": "1280px",
          "padding": ".75rem .875rem",
          "radius": ".5rem",
          "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
          "size": "16px",
        },
        "transition": Object {
          "time": ".3s ease",
        },
      }
    }
  >
    <StyledComponent
      forwardedComponent={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bdVaJa",
            "isStatic": false,
            "lastClassName": "cTLxvN",
            "rules": Array [
              "
all: unset;
position: relative;
cursor: pointer;
height: 1.3rem;
width: 2rem;
padding: 0;
> p {
  all: unset;
  transition: ",
              [Function],
              ";
  border: ",
              [Function],
              ";
  border-color: ",
              [Function],
              ";
  transform-origin: center;
  border-radius: .125rem;
  display: inline-block;
  position: absolute;
  width: 100%;
  content: '';
  &:nth-child(1) {
    transform: rotate(",
              [Function],
              ");
    top: calc(",
              [Function],
              " - .125rem);
  }
  &:nth-child(2) {
    transform: scaleX(",
              [Function],
              ");
    top: calc(50% - .1rem);
  }
  &:nth-child(3) {
    transform: rotate(",
              [Function],
              ");
    bottom: calc(",
              [Function],
              " - .125rem);
  }
}
",
            ],
          },
          "defaultProps": Object {
            "theme": Object {
              "animation": [Function],
              "color": Object {
                "black": "#222222",
                "darkBlue": "#002C5F",
                "darkGray": "#515151",
                "error": "#dd3333",
                "gray": "#a9abae",
                "lightGray": "#e5e5e5",
                "overlay": "rgba(0, 0, 0, .15)",
                "primary": "#00528F",
                "secondary": "#00245f",
                "success": "#33bb33",
                "warn": "#dd3333",
                "white": "#fefefe",
              },
              "font": Object {
                "color": "#222222",
                "family": "\\"Ubuntu\\", sans-serif",
                "size": "16px",
                "spacing": 0,
              },
              "shape": Object {
                "border": ".1rem solid rgba(0, 0, 0, .15)",
                "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
                "margin": ".25rem .125rem",
                "maxWidth": "1280px",
                "padding": ".75rem .875rem",
                "radius": ".5rem",
                "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
                "size": "16px",
              },
              "transition": Object {
                "time": ".3s ease",
              },
            },
          },
          "displayName": "styled.button",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-bdVaJa",
          "target": "button",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
      open={false}
      theme={
        Object {
          "animation": [Function],
          "color": Object {
            "black": "#222222",
            "darkBlue": "#002C5F",
            "darkGray": "#515151",
            "error": "#dd3333",
            "gray": "#a9abae",
            "lightGray": "#e5e5e5",
            "overlay": "rgba(0, 0, 0, .15)",
            "primary": "#00528F",
            "secondary": "#00245f",
            "success": "#33bb33",
            "warn": "#dd3333",
            "white": "#fefefe",
          },
          "font": Object {
            "color": "#222222",
            "family": "\\"Ubuntu\\", sans-serif",
            "size": "16px",
            "spacing": 0,
          },
          "shape": Object {
            "border": ".1rem solid rgba(0, 0, 0, .15)",
            "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
            "margin": ".25rem .125rem",
            "maxWidth": "1280px",
            "padding": ".75rem .875rem",
            "radius": ".5rem",
            "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
            "size": "16px",
          },
          "transition": Object {
            "time": ".3s ease",
          },
        }
      }
    >
      <button
        className="sc-bdVaJa cTLxvN"
        open={false}
      >
        <p />
        <p />
        <p />
      </button>
    </StyledComponent>
  </styled.button>
</MenuButton>
`;
